src/index.js

```diff
@@ -17,8 +27,19 @@ app.get('/test', (req, res) => {
   res.json({ status: 'Working!' });
 });
 
+app.get('/checkdb', async (req, res) => {
```

Comment:
@Saumya29 this one is used for?

Reply:
I used it to test if Mongo is running correctly. Will remove.